test/aspect.spec.js

```diff
@@ -47,15 +45,24 @@ describe('Aspect Functions', () => {
     expect(g.state.errors[1].message).to.equal('Concurrent Runs Not Allowed')
   })
   it('should support throwing in onError', async () => {
+    // Use the single error object to avoid 'Unhandled promise rejection' in
+    // some browsers.
+    let theException = new Error('hi from aspect')
     let ThrowHi = aspect({
       onError: () => {
-        throw Error('hi from aspect')
+        throw theException
       },
     })
     let throwsHi = ThrowHi(() => {
       throw Error('Not hi')
     })
-    expect(throwsHi()).to.be.rejectedWith(Error('hi from aspect'))
+
+    // Replace chai-as-promised with more traditional try/catch approach.
+    try {
+      await throwsHi()
```

Comment:
you should throw a different exception at this line because if it _doesn't_ throw an exception, the test will still pass

Reply:
That's a good catch.  Something like this should be fine?

`expect(false).to.equal(true)`

Or I can bring back the `chai-as-promised`.